src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const th = table.tHead.rows[0].cells[1];
+const targetTh = table.tHead.rows[0].cells[4];
+const newTh = table.tHead.rows[0].insertCell(5);
+
+newTh.outerHTML = targetTh.outerHTML;
+targetTh.innerHTML = th.innerHTML;
+
+const tf = table.tFoot.rows[0].cells[1];
+const targetTf = table.tFoot.rows[0].cells[4];
+const newTf = table.tFoot.rows[0].insertCell(5);
+
+newTf.outerHTML = targetTf.outerHTML;
+targetTf.innerHTML = tf.innerHTML;
+
+for (let row = 0; row < table.tBodies[0].rows.length; row++) {
```

Comment:
Iterating over rows with hardcoded cell indices can lead to errors if the table structure changes. Consider using a more dynamic approach to identify the target cells.